Forms/MainForm.cs

```diff
@@ -361,13 +410,30 @@ private void txtCharacterList_TextChanged(object sender, EventArgs e)
             CharacterList charList = new CharacterList();
             charList.AddRange(txtCharacterList.Text.ToArray());
             Configuration.DefaultCharacters = charList;
+            LoadConfigurationDetails();
         }
 
         private void txtWordList_TextChanged(object sender, EventArgs e)
         {
+            int oldWordCount = Configuration.DefaultWords.Count;

```

Comment:
Maybe...but I'd guess normally people will copy-paste into the text field, so they won't be typing spaces. Also, after entering the last word, there won't be an update until losing focus.

An update ONLY on losing focus might work without being confusing; but the user would need to select a new text box, plus they won't see the strength bar updating as they add words.

Or do you have other ideas?